packages/components/src/components/Image/images.ts

```diff
@@ -62,4 +62,6 @@ export const PNG_IMAGES = {
     COINJOIN_MESS: 'CoinjoinMess.png',
     DONT_DISCONNECT: 'DontDisconnect.png',
     DONT_DISCONNECT_2x: 'DontDisconnect@2x.png',
+    SHIELD: 'Shield.png',
```

Comment:
I think we have those images already in `suite-data` => `Check.png`

Reply:
0c4db648f